src/DSP/FFT/FFT.cpp

```diff
@@ -71,99 +37,6 @@ void FFT::ConvertToDBFS(std::vector<float>& bins)
         amplitude = amplitude > 0 ? 10 * log10(amplitude) : -150;
 }
 
-void FFT::Calculate(const complex16_t* src, uint32_t count, std::vector<float>& outputBins)
-{
-    assert(count == m_fftCalcIn.size());
-    for (uint32_t i = 0; i < count; ++i)
-    {
-        m_fftCalcIn[i].r = src[i].real() / 32768.0 * mWindowCoefs[i];
-        m_fftCalcIn[i].i = src[i].imag() / 32768.0 * mWindowCoefs[i];
-    }
-    kiss_fft(m_fftCalcPlan, m_fftCalcIn.data(), m_fftCalcOut.data());
-    outputBins.resize(count);
-    for (uint32_t i = 0; i < count; ++i)
-    {
-        float amplitude = (m_fftCalcOut[i].r * m_fftCalcOut[i].r + m_fftCalcOut[i].i * m_fftCalcOut[i].i) / (count * count);
-        outputBins[i] = amplitude > 0 ? 10 * log10(amplitude) : -150;
-    }
-}
-
-void FFT::Calculate(const complex32f_t* src, uint32_t count, std::vector<float>& outputBins)
-{
-    assert(count == m_fftCalcIn.size());
-    for (uint32_t i = 0; i < count; ++i)
-    {
-        m_fftCalcIn[i].r = src[i].real() * mWindowCoefs[i];
-        m_fftCalcIn[i].i = src[i].imag() * mWindowCoefs[i];
-    }
-    kiss_fft(m_fftCalcPlan, m_fftCalcIn.data(), m_fftCalcOut.data());
-    outputBins.resize(count);
-    for (uint32_t i = 0; i < count; ++i)
-    {
-        float amplitude = (m_fftCalcOut[i].r * m_fftCalcOut[i].r + m_fftCalcOut[i].i * m_fftCalcOut[i].i) / (count * count);
-        outputBins[i] = amplitude > 0 ? 10 * log10(amplitude) : -150;
-    }
-}
-
-void FFT::ProcessLoop()
```

Comment:
This functionality should not be removed, it is intended to be reusable replacement for logic hardcoded into GUI
https://github.com/Matom-ai/LimeSuiteNG/blob/9e86b01433e66c04c2697158f2fac7c36c588476/GUI/fftviewer_wxgui/fftviewer_frFFTviewer.cpp#L529-L604